binding/tfjs_backend.h

```diff
@@ -20,15 +20,18 @@
 
 #include <node_api.h>
 #include <map>
+#include <memory>
 #include <string>
 #include "../deps/tensorflow/include/tensorflow/c/eager/c_api.h"
 
 namespace tfnodejs {
 
 class TFJSBackend {
  public:
-  // TODO(kreeger): Move to a factory and make this private.
-  TFJSBackend(napi_env env);
+  // Creates, initializes, and returns a TFJSBackend instance. If initialization
+  // fails, a nullptr is returned.
+  static std::unique_ptr<TFJSBackend> Create(napi_env env);
+
   ~TFJSBackend();
```

Comment:
The destructor should be moved to `private` as well to prevent it from being constructed outside of the class.

Reply:
Done. The destructor was reqired to be public for unique_ptr.